idaes/models/unit_models/solid_liquid/tests/test_thickener.py

```diff
@@ -39,44 +38,43 @@
     StateBlockData,
     StateBlock,
     LiquidPhase,
+    SolidPhase,
     Component,
 )
 from idaes.models.unit_models.solid_liquid import Thickener0D
+from idaes.core.solvers import get_solver
+from idaes.core.initialization import (
+    BlockTriangularizationInitializer,
+    InitializationStatus,
+)
+from idaes.core.util import DiagnosticsToolbox
 from idaes.models.unit_models.separator import (
-    Separator,
     SeparatorData,
     EnergySplittingType,
 )
 from idaes.core.util.model_statistics import (
-    degrees_of_freedom,
     number_variables,
     number_total_constraints,
     number_unused_variables,
 )
-from idaes.core.solvers import get_solver
-from idaes.core.initialization import (
-    BlockTriangularizationInitializer,
-    InitializationStatus,
-)
-from idaes.core.util import DiagnosticsToolbox
 
 # -----------------------------------------------------------------------------
 # Get default solver for testing
 solver = get_solver()
 
 
-@declare_process_block_class("TestParameterBlock")
-class TestParameterData(PhysicalParameterBlock):
+@declare_process_block_class("SolidParameterBlock")
+class SolidParameterData(PhysicalParameterBlock):
     def build(self):
         """
         Callable method for Block construction.
         """
         super().build()
 
-        self._state_block_class = TestStateBlock
+        self._state_block_class = SolidStateBlock
 
         # Add Phase objects
-        self.Liq = LiquidPhase()
+        self.sol = SolidPhase()
```

Comment:
```suggestion
        self.Sol = SolidPhase()
```
Minor comment, but isn't this usually capitalized?

Reply:
It is a name, so it doesn't really matter (plus it is just for a test).